submission/persistence-filesystem/src/main/kotlin/ac/uk/ebi/biostd/persistence/filesystem/pagetab/PageTabUtil.kt

```diff
@@ -20,28 +26,49 @@ class PageTabUtil(
     private val fileListMapper: ToFileListMapper,
 ) {
     suspend fun generateSubPageTab(
-        sub: ExtSubmission,
+        extSub: ExtSubmission,
         target: File,
     ): PageTabFiles {
-        val element = toSubmissionMapper.toSimpleSubmission(sub)
-        val permissions = sub.permissions()
+        val sub = toSubmissionMapper.toSimpleSubmission(extSub)
+
+        if (extSub.released.not() && sub.reviewType == DOUBLE_BLIND) {
+            sub.section = filterBlindReview(sub.section) ?: Section()
```

Comment:
This logic looks good but I think it does not bellow here, if you notice is in the toSimpleSubmission method that we filter the values not required by simple submission format, like removing public access tag. So I think we should keep that logic there.

WDYT?